docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/DetectChangePointBySsaBatchPrediction.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class DetectChangePointBySsaBatchPrediction
```

Comment:
>DetectChangePointBySsaBatchPrediction [](start = 24, length = 37)

They are not referenced from anywhere. 
DO you think they would be better suited for the machinelearning-samples repo?

cc @shmoradims

Reply:
I’ll reference them from prediction engine. We have had customers ask us about this.